compile.sh

```diff
@@ -34,73 +34,73 @@ function checkFiles() {
 }
 
 echo "[compile] linux arm";
-env GOOS=linux GOARCH=arm go build;
+env GOOS=linux GOARCH=arm go build -ldflags="-s -w" -gcflags=-trimpath="${GOPATH}" -asmflags=-trimpath="${GOPATH}";
```

Comment:
out of curiosity, what's the effect of these changes?

Reply:
These flags strip debugging symbols and remove the GOPATH from callstack traces.

This reduces the binary file size and removes any references to my local machine such as paths.